src/components/Footer.jsx

```diff
@@ -1,20 +1,175 @@
 import React from 'react'
+import { Link } from 'gatsby'
+import slackLogo from '../images/slack.svg'
+import githubLogo from '../images/github.svg'
+import meetup from '../images/meetup.svg'
 import styled from 'styled-components'
-import logo from '../images/cfc_logo_short.svg'
+import longArrowAltRight from '../images/join-us/long-arrow-alt-right.svg'
 
 const StyledFooter = styled.footer`
-  width: 14rem;
+  width: 100%;
+  padding: 1rem 0;
+  display: block;
+  margin: 0 auto;
+
+  @media (min-width: 1200px) {
+    padding: 0rem 15.25rem;
+  }
+
+  .footer-content {
+    margin: 0 auto;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    gap: 1rem;
```

Comment:
It looks like the gap is 2rem in the Figma. 
<img width="815" alt="Screen Shot 2023-01-15 at 7 20 38 PM" src="https://user-images.githubusercontent.com/6326891/212579438-1ad2cac1-cb4c-468f-92d8-bae899b37104.png">